src/classes/RD2_ElevateIntegrationService.cls

```diff
@@ -260,150 +255,15 @@ public inherited sharing class RD2_ElevateIntegrationService {
         }
     }
 
-
     /**
     * @description Cancels commitments in Elevate connected to the closed Recurring Donations
     * @param closedRDs Closed RDs with the user specified closed reason
     * @return void
     */
-    public void cancelCommitments(List<npe03__Recurring_Donation__c> closedRDs) {
-        new CancelCommitmentService().cancelCommitments(closedRDs);
+    public void cancelCommitments(List<RD2_CancelCommitmentService.RecordWrapper> closedRDs) {
```

Comment:
I think this method is redundant and will be better to completely remove this method and call 
`new RD2_CancelCommitmentService().cancelCommitments(closedRDs);` in the trigger.